### PR TITLE
Healthcare API

### PR DESCRIPTION
**Pull request by joaodaher**

### CODE REVIEW

File: gcp_pilot/healthcare.py [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+    def __init__(self, **kwargs):
+        super().__init__(
+            serviceName="healthcare",
+            version="v1beta1",
+            cache_discovery=False,
+            **kwargs,
+        )
+
+    def _set_location(self, location: str = None) -> str:
+        # Only some limited regions are available: https://cloud.google.com/healthcare-api/docs/concepts/region
```
Comment (guilhermearaujo) [inline gcp_pilot/healthcare.py:21]:
```suggestion
        # Only some limited regions are available: https://cloud.google.com/healthcare-api/docs/concepts/regions
````

File: gcp_pilot/healthcare.py [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+        return "us"
```
Comment (guilhermearaujo) [inline gcp_pilot/healthcare.py:22]: Wouldn't it be better to `return location` and set the default value to `"us"`?

Comment (joaodaher) [reply gcp_pilot/healthcare.py:22]: Sure, I can set the default only.

File: gcp_pilot/healthcare.py (hunk 1 above)
Comment (joaodaher) [reply gcp_pilot/healthcare.py:21]: Thank you

**Pull request merged**